modules/helm-creating-a-custom-helm-chart-on-openshift.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="creating-a-custom-helm-chart-on-openshift_{context}"]
+= Creating a custom Helm chart on OpenShift
+
+. Create a new project:
++
+----
+$ oc new-project nodejs-ex-k
+----
+
+. Download an example Node.js chart that contains OpenShift objects:
++
+----
+$ git clone https://github.com/redhat-developer/redhat-helm-charts/tree/tech-preview-4.3/alpha/nodejs-ex-k
+----
+
+. Go to the directory with the sample chart:
++
+----
+$ cd redhat-helm-charts/alpha/nodejs-ex-k/
+----
+
+. Edit the `Chart.yaml` and add a description of your chart:
++
+[source,yaml]
+----
+apiVersion: v2
+name: Node.js
+description: A Helm chart for OpenShift
+----
+
+. Verify that the chart is formatted properly:
++
+----
+$ helm lint
+----
+
+. Install the chart:
++
+----
+$ cd ..
+$ helm install nodejs-chart nodejs-ex-k
+----
+
+. Verify that the chart has installed successfully:
++
+----
+$ helm list
```

Comment:
Same about adding what the expected output is for it being successful.

Reply:
Added